### DISCUSSION TITLE
2.0图形客户端用什么软件运行比较好

### DISCUSSION THREAD

**Discussion by CleverMod** [Ljzd-PRO/Mys_Goods_Tool#134] [Q&A]
win10直接双击运行展示效果不太好, 卡顿比较严重

**Comment by Ljzd-PRO** [comment]
Windows 10 的话，可以试试在微软应用商店安装Windows Terminal，微软做的终端软件，Windows 11自带

**Comment by HA-JD** [comment]
整成网页端比较好

**Comment by CleverMod** [comment]
> Windows 10 的话，可以试试在微软应用商店安装Windows Terminal，微软做的终端软件，Windows 11自带

win10无法降windows terminal设置成默认终端咋办额..

**Comment by Ljzd-PRO** [comment]
> > Windows 10 的话，可以试试在微软应用商店安装Windows Terminal，微软做的终端软件，Windows 11自带
> 
> win10无法降windows terminal设置成默认终端咋办额..

在Windows Terminal通过命令来启动程序。
Windows Terminal 进入到程序所在目录，然后
```powershell
.\Mys_Goods_Tool
```

**Comment by Ljzd-PRO** [comment]
网页端的话暂时不会有，不过Textual说是未来版本会支持网页显示

**Comment by CleverMod** [comment]
> 

谢谢 有用

**Comment by Tsuk1ko** [comment]
由于我将软件放在一台 linux 上（在局域网内）

用 Xshell 6 连接过去启动，界面显示有点异常并且直接卡死

用 Terminal 启动 wsl 然后 ssh 过去，界面显示正常，可以操作，但有点卡顿，并且过了一小会也会卡死😂

**Comment by Ljzd-PRO** [comment]
> 由于我将软件放在一台 linux 上（在局域网内）
> 
> 用 Xshell 6 连接过去启动，界面显示有点异常并且直接卡死
> 
> 用 Terminal 启动 wsl 然后 ssh 过去，界面显示正常，可以操作，但有点卡顿，并且过了一小会也会卡死😂

如果能用Windows Terminal运行会比较好。也可以在本地用Windows Terminal创建好兑换计划，把`user_data.json`拷贝过去，然后远程主机加上运行参数`-m exchange-simple`在无交互UI的情况下启动兑换模式

wsl的话，如果是用Windows Terminal连接，我的是能正常运行，用的是Ubuntu 22.04
